collector/nodes.go

```diff
@@ -96,6 +96,7 @@ var (
 	defaultRoleLabels               = []string{"cluster", "host", "name"}
 	defaultThreadPoolLabels         = append(defaultNodeLabels, "type")
 	defaultBreakerLabels            = append(defaultNodeLabels, "breaker")
+	defaultIndexingPressureLabels   = append(defaultNodeLabels, "indexing_pressure")
```

Comment:
Can we remove the use of defaultNodeLabels and instead only include cluster, host, and name? The others should be removed long term because there are more space efficient ways of getting to that data query time.